intelmq/bin/intelmqctl.py

```diff
@@ -117,33 +118,37 @@ def __init__(self, runtime_configuration, logger, controller):
                 self.logger.error('Directory %s does not exist and cannot be '
                                   'created: %s.', self.PIDDIR, exc)
 
-    def bot_run(self, bot_id):
+    def bot_run(self, bot_id, run_subcommand=None, console_type=None, message_action_kind=None, dryrun=None, msg=None):
         pid = self.__read_pidfile(bot_id)
-        if pid:
-            if self.__status_process(pid):
-                log_bot_error('running', bot_id)
-                return 'running'
-            else:
-                self.__remove_pidfile(bot_id)
-        log_bot_message('starting', bot_id)
-        filename = self.PIDFILE.format(bot_id)
-        with open(filename, 'w') as fp:
-            fp.write(str(os.getpid()))
+        if pid and self.__status_process(pid):
+            self.logger.warning("Main instance of the bot is running in the background and will be stopped; "
+                                "when finished, we try to relaunch it again."
```

Comment:
Where please? "and will be stopped; **SPACE** when finished"
I must be blind :D